src/runtime/virtcontainers/clh.go

```diff
@@ -287,12 +287,14 @@ func (clh *cloudHypervisor) createSandbox(ctx context.Context, id string, networ
 
 	// set the serial console to the cloud hypervisor
 	if clh.config.Debug {
-		clh.vmconfig.Serial = chclient.NewConsoleConfig(cctTTY)
+		clh.vmconfig.Console = chclient.NewConsoleConfig(cctTTY)
+		clh.vmconfig.Console.SetMode("Tty")
```

Comment:
Isn't it redundant to create a new console config with `Tty` and then set the mode to `Tty`?

Reply:
Fixed.